app/templates/gulp/gulp/config.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var fs = require( 'fs' );
+
+/**
+ * Set up file paths
+ */
+var loc = {
+  src:  './src',
+  dist: './dist',
+  lib:  JSON.parse( fs.readFileSync( './.bowerrc' ) ).directory, // eslint-disable-line no-sync, no-inline-comments, max-len
+  test: './test'
+};
+
+module.exports = {
+  pkg:    JSON.parse( fs.readFileSync( 'bower.json' ) ), // eslint-disable-line no-sync, no-inline-comments, max-len
+  banner:
+      '/*!\n' +
+      ' *  <%%= pkg.name %> - v<%%= pkg.version %>\n' +
+      ' *  <%%= pkg.homepage %>\n' +
+      ' *  Licensed <%%= pkg.license %> by <%= pkg.author.name %> <%= pkg.author.email %>\n' +
+      ' */',
+  lint: {
+    src: [
+      loc.src + '/static/js/**/*.js',
+      loc.test + '/unit_tests/**/*.js',
+      loc.test + '/browser_tests/**/*.js'
+    ],
+    gulp: [
+      'gulpfile.js',
+      'gulp/**/*.js'
+    ]
+  },
+  test: {
+    src:   loc.src + '/static/js/**/*.js',
+    tests: loc.test
+  },
+  clean: {
+    dest: loc.dist
+  },
+  styles: {
+    cwd:      loc.src + '/static/css',
+    src:      '/main.less',
+    dest:     loc.dist + '/static/css',
+    settings: {
+      paths: [
+        loc.lib,
+        loc.lib + '/cf-typography/src'
+      ],
+      compress: true
+    }
+  },
+  scripts: {
+    src: [
+      loc.lib + '/jquery/dist/jquery.js',
+      loc.lib + '/jquery.easing/js/jquery.easing.js',
```

Comment:
Oh, now I'm not sure, cause it was in [the Gruntfile](https://github.com/jimmynotjim/generator-cf/blob/gulp-option/app/templates/grunt/_Gruntfile.js#L46)